modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -12,7 +12,7 @@ To manage the volume of traffic going through a broker, Redpanda implements thro
 
 With Redpanda's xref:get-started:architecture.adoc#thread-per-core-model[thread-per-core model], the Kafka API traffic to and from a client connection is processed by a single core (shard). In order to manage throughput quotas efficiently, broker quotas are distributed between shards, and each per-shard quota is in turn shared by all connections served by the shard. Splitting broker quota optimally between shards is done behind the scenes by the quota balancer component.
 
-To distribute the broker throughput quota, the balancer periodically monitors the throughput rate of a broker's shards, and it distributes more quota to the shards that can make better use of it than the others. Each shard has a minimum throughput quota value, which is configurable both as a percentage of the default quota and as an absolute rate limit.
+To distribute the broker throughput quota, the balancer periodically monitors the throughput rate of a broker's shards, and it distributes more quota to the shards that can make better use of it than the others.
```

Comment:
The quota balancer is not in use when v2 is enabled; all shards share a single token bucket (well, one for ingress and one for egress). I'd be tempted to drop both of these paragraphs.